packages/forms/src/UIForm/fieldsets/Array/DefaultArrayTemplate.component.js

```diff
@@ -25,7 +25,11 @@ export function DefaultArrayTemplate(props) {
 			/>
 			<ol id={id} className={classNames(theme['tf-array'], 'tf-array')}>
 				{value.map((itemValue, index) => (
-					<li className={theme.item} key={index}>
+					<li
+						className={classNames(theme.item, 'item', `item-${index}`)}
+						key={index}
+						id={`tf-array-item-${index}`}
```

Comment:
If you have 2 arrays you'll have id collision.
Can you include the props `id` which is specific to the property please ?

Reply:
thx for the catch